### ISSUE TITLE
FakeReporter

### ISSUE THREAD

**Issue by haykam** [Wurst-Imperium/Wurst-Client#721]
A feature where clicking on its name in the menu will bring up a box saying "Make a fake message in chat (color codes supported) and then typing it in will make the message (with colors) into the chat log. Then, the Wurst bar and all other signs of hacking will be removed (like VideoSafe option that I'm also putting as a suggestion) and the screenshot will be taken and saves to your desktop or screenshots folder.

**Comment by GummiBoat** [comment]
a) It's not possible to send Color Codes (the § Symbol) as a Client. (You'll get disconnected, if you try to.)
b) You can't send chat messages from other people.
c) No matter how often you ask, there will be no option to remove the Wurst-"Bar" from your screen. (Press F1 if you really need to.)

Or if I understood it wrong and you mean that the Message is only send Client-Side and only in your Chat visible, there are also some flaws with it.
First -> You have to go crazy and add everything that's within the Name too. Prefixes, Suffixes, Channel-Names from Plugins, Factions, etc.
Second -> No one will ban someone who wrote bullshit in chat, just because you send a screenshot of it to him. (Especially if you're the only one that can provide that information.)

**Comment by haykam** [comment]
How's this invalid?

**Comment by OMGchangeMYusername** [comment]
you suggested multiple features in the same issue,and (for me) your description was a little confusing tbh...

**Comment by haykam** [comment]
It's one feature.

**Comment by OMGchangeMYusername** [comment]
well to be even more honest,it doesn't worth the effort.. 
like who wants to do that to someone?did the guy kill your gf?
and "making" this is on a whole different level...

**Comment by haykam** [comment]
Wow. Just leave me alone. Bitch.

**Comment by OMGchangeMYusername** [comment]
idk where did you see a bitch here but i can understand how difficult is life for you.. never give up!

**Comment by haykam** [comment]
A feature where clicking on its name in the menu will bring up a box saying "Make a fake message in chat (color codes supported) and then typing it in will make the message (with colors) into the chat log. Then, the Wurst bar and all other signs of hacking will be removed (like VideoSafe option that I'm also putting as a suggestion) and the screenshot will be taken and saves to your desktop or screenshots folder.
-------------------------------------------------------------------------------------------------------------------------------------------
ORIGINAL ABOVE | EDITED BELOW
-------------------------------------------------------------------------------------------------------------------------------------------
Basically, this will be a new mod called FakeReporter where you will be able to type a client-side message, similar to /tellraw. It will then put it in chat. Finally, Panic will turn on and the Wurst bar will hide. You can then take a screenshot and once it detects F2 is pressed, all mods turned on before will be back on and the Wurst bar will be visible again.
